<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchMusicListItem</class>
 <widget class="QWidget" name="SearchMusicListItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>101</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#lblTitle {
font: 16px &quot;Arial&quot;;
font-weight: bold;
color: rgb(248, 198, 59);
}

#lblArtist {
font: 16px &quot;Arial&quot;;
font-weight: bold;
color: rgb(234, 160, 94);
}

#lblAlbum {
font: 16px &quot;Arial&quot;;
font-weight: bold;
color: rgb(250, 180, 110);
}

#cmbAddTo {
border-image: url(:/theme/searchmusic/RES/theme/Musik_Suche/Dropdown_normal.png);
padding-right: 25px;
padding-left: 10px;
}

#cmbAddTo::drop-down {
border-width: 0px;
}

#cmbAddTo:hover {
border-image: url(:/theme/searchmusic/RES/theme/Musik_Suche/Dropdown_hover.png);
}

#cmbAddTo:on {
border-image: url(:/theme/searchmusic/RES/theme/Musik_Suche/Dropdown_pressed.png);
}

#lblBackground {
background-image: url(:/theme/searchmusic/RES/theme/Musik_Suche/slot_normal.png);
}
#lblBackground:hover {
background-image: url(:/theme/searchmusic/RES/theme/Musik_Suche/slot_hover.png);
}
QPushButton {
font: 14px;
font-weight: bold;
font-family: arial, verdana, sans-serif;
color: rgb(255,255,255);
border: none;
background-repeat: none;
background-image: url(:/theme/searchmusic/RES/theme/Musik_Suche/Download_Button_normal.png);
}

QPushButton:hover {
background-image: url(:/theme/searchmusic/RES/theme/Musik_Suche/Download_Button_hover.png);
}

QPushButton:pressed {
background-image: url(:/theme/searchmusic/RES/theme/Musik_Suche/Download_Button_pressed.png);
}</string>
  </property>
  <widget class="QLabel" name="lblTitleCaption">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>32</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Titel:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>19</y>
     <width>138</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Levels</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblArtistCaption">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>43</y>
     <width>39</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Artist:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblArtist">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>43</y>
     <width>138</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Avicii</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbAddTo">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>13</y>
     <width>163</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblBackground">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>374</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPlay">
   <property name="geometry">
    <rect>
     <x>261</x>
     <y>56</y>
     <width>104</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblAlbumCaption">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>67</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Album:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblAlbum">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>67</y>
     <width>141</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Levels</string>
   </property>
  </widget>
  <zorder>lblBackground</zorder>
  <zorder>lblTitleCaption</zorder>
  <zorder>lblTitle</zorder>
  <zorder>lblArtistCaption</zorder>
  <zorder>lblArtist</zorder>
  <zorder>cmbAddTo</zorder>
  <zorder>btnPlay</zorder>
  <zorder>lblAlbumCaption</zorder>
  <zorder>lblAlbum</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
